.github-labelsrc.json

```
@@ -1,4 +1,10 @@
 [
+  {
+    "id": 4294819708,
+    "name": "⌛ Status: Waiting for Customer",
```

I don't think the author of the issue necessarily needs to be the customer. 
I feel like we can use `user` instead of `customer`. 

I wonder what the rest of the team thinks about this though.

---

We can keep it a bit more generic:

```suggestion
    "name": "⌛ Status: Awaiting for feedback",
```

---

Updated in this commit: 5bada16b028c5bdb1af5e803162edc1c44c496f3